MauroDataMapper/mdm-core
# Feature/gh 463 version folder finalised merge

Updated the simpleModelVersionTree() method in the versionFolderService to take the branchesOnly and forMerge parameters in the same manor as the method in the ModelService

closes #463 

to Test:
Use this branch
In mdmUI create a versionfolder with something in it.
finalise it.
create a new version 1 (draft)
create a new version 2 (draft)
create a fork (draft)

in version 1 click merge on the expanded menu
![image](https://github.com/user-attachments/assets/551da2c2-6286-41b3-85ba-b382eadc79c5)
Note that the finalised version no longer shows up
